integration/rpctest/btcd.go

```diff
@@ -0,0 +1,71 @@
+// Copyright (c) 2017 The btcsuite developers
+// Use of this source code is governed by an ISC
+// license that can be found in the LICENSE file.
+
+package rpctest
+
+import (
+	"fmt"
+	"go/build"
+	"os/exec"
+	"path/filepath"
+	"runtime"
+	"sync"
+)
+
+var (
+	// btcdCompileMtx guards access to the btcd path so that it is only compiled
+	// once.
+	btcdCompileMtx sync.Mutex
```

Comment:
Similarly, I think `compileMtx` would be better here since it's agnostic for forks.